README.Rmd

```diff
@@ -114,15 +114,12 @@ starwars %>%
 you control what goes in the nested column. `Z` is `N` but `r emo::ji("arrow_heading_down")`. 
 
 ```{r}
-tbl <- tibble(cyl = c(4, 6, 8), mpg = c(30, 25, 20)) 
+tbl <- tibble(cyl_threshold = c(4, 6, 8), mpg_threshold = c(30, 25, 20)) 
 tbl %>%
-  zest_join(mtcars, data = ~cyl == !!cyl & mpg < !!mpg)
+  zest_join(mtcars, data = ~cyl == !!cyl_threshold & mpg < !!mpg_threshold)
```

Comment:
same as above. No need for the `!!` here since there is no ambiguity.

Reply:
The !! are not needed here, that’s another bonus from using different names as you did.